packages/core-api/lib/versions/2/handlers/transactions.js

```diff
@@ -41,7 +41,7 @@ exports.store = {
    */
   async handler (request, h) {
     if (!transactionPool.options.enabled) {
-      return Boom.teapot()
+      return Boom.teapot('Transaction pool is disabled.')
```

Comment:
I said `serverUnavailable` in my previous comment, not `teapot` as that doesn't give a clear indication of what is wrong just based on the status code.

Reply:
done